database/seeders/UserSeeder.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Seeders;
+
+use App\Models\User;
+use Illuminate\Database\Seeder;
+
+class UserSeeder extends Seeder
+{
+    public function run(): void
+    {
+        User::factory()->create()->assignRole("super-admin");
+        User::factory()->create()->assignRole("admin");
+        User::factory()->create()->assignRole("user");
```

Comment:
Lets create in `UserFactory` methos like `admin()` to assign `role` after creating.
Example: https://github.com/blumilksoftware/toby/blob/9d15ffea8ea64322f16265a9e669ca44db352eb8/database/factories/UserFactory.php#L37
but in that example permissions are synced, we must assign role.

Reply:
Done.